config/styleguide.config.js

```diff
@@ -324,6 +324,14 @@ module.exports = {
         fontSize: 'inherit',
       },
     },
+    StyleGuide: {
+      '@global body *':
+        styleguidistEnv === 'e2e'
+          ? {
+              animationPlayState: 'paused !important',
+            }
+          : {},
+    },
```

Comment:
This seems a bit odd to me to add a global style like this using the StyleGuide component, but I see what you are trying to do.

Perhaps I can recommend an alternative:

Use the `require` config option to dynamically include an extra CSS file with the above styles.

```css
// e2e.css

body * {
  animationPlayState: paused !important;
}
```

```js
// styleguide.config.js

{
  require: [
    path.resolve('packages/css-reset/index.scss'),
    path.resolve('docs/scss/styleguide.scss'),
    styleguideEnv === 'e2e' && path.resolve('docs/scss/e2e.css')
  ]
}
```

I think this is a bit cleaner and clearer.

Reply:
When I do it this way, all the baseline images change again. I suspect it has something to do with a race condition of having e2e.css getting injected into the `<head>` causing the CSSOM to reset and cause a flash of unstyled fonts - which changes the kerning of some text in the new baseline images.

How about we put this out of scope along with your next suggestion? That way we can add a longer timeout period before creating screenshots.

Reply:
I'm fine with that, though I don't know about the race condition you outline. Either way, using the `theme` or `require`, the styles are being injected into the <head> at runtime. Seems that using the 2 strategies slightly changes the overall page styling somehow though.